### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewExplorer_ScreenReader: Missing grouping for listbox items within "Open Editors" and "File Explorer" section.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52407]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : HighSierra10.13.5

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader : VoiceOver

**Repro Steps:**
1. Launch VS Code while VoiceOver is ON.
2. Navigate to Activity Bar and select "Explorer" button.
3. Navigate to "Open Editors" and "File Explorer" section listbox items using "Tab".
4. Read all the listbox items.

**Actual:**
VoiceOver does not read out the total no of listbox items as well as its ordering(1/total,2/total and so on) when navigated using downarrow key within items. 

**Expected:**
VoiceOver should read total no of listbox items when screenreader lands onto the first listbox item as well as its ordering with each of them.

**Recommendations:**
Manage the content within fieldset and legend tags for grouping listbox items.
or,
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The screenreader users will not be able to know the total count of listbox items present as well as which item is screenreader reading out of the total items present. This information can be useful for navigating within large listbox items.

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment For Reference:**
[A11y_VSCode_ViewExplorer_VoiceOver_FileExplorerListbox.pptx](https://github.com/Microsoft/vscode/files/2118269/A11y_VSCode_ViewExplorer_VoiceOver_FileExplorerListbox.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/52008
